docs/Banners.md

```diff
@@ -0,0 +1,45 @@
+# Banners
+
+The HCMI Portal UI supports displaying informational banners at the top of each page that can be configured through environment variables.
+
+![Banners example](./images/banners.png "Banners example")
+
+## Usage
+
+To configure banners, provide an array of JSON objects representing the contents of each banner in the `REACT_APP_BANNERS` environment variable for the UI project (`ui/.env`). Multiple banners can be present at once.
+
+### Banner JSON
+
+Each object should be structured as follows:
+```json
+{
+  "title": string,
+  "message": string,
+  "type": INFO | WARNING | CRITICAL,
+  "dismissible": boolean,
+}
+```
+
+The object properties are as follows:
```

Comment:
> https://github.com/nci-hcmi-catalog/portal/pull/1078/files#r1887067553

not for this PR but as my note-to-self for Stage: this part here could benefit from having a visual "anatomical" image, showing which one is the title, which the message, etc. as banners could have additional features (was it VirusSeq?). and which is Info vs Warning, and how the "Dismissable" changes things (including banner expiration and scheduled frequency, etc.)